### DISCUSSION TITLE
Custom certificates

### DISCUSSION THREAD

**Discussion by dib0** [docusealco/docuseal#24] [General]
Hi,

Quick question: is it possible to use my own (for example letsencrypt or bought) certificates? In PEM of PKCS12 format?

Looking through the repo I see the environment variable CERT in [lib/docuseal.rb](https://github.com/docusealco/docuseal/blob/a92d1d82c1249ecb1db596b2d13814dd416675e5/lib/docuseal.rb#L14)
```
CERTS = JSON.parse(ENV.fetch('CERTS', '{}'))
```

That seems to have a json format. Should I use this? Or can I point to a file?

Is this already possible? And, if so, how should I do this?

**Comment by AlexBTurchyn** [comment]
@dib0 We have recently introduced the ability to upload custom certificates. You can utilize [this page](https://docuseal.co/settings/esign) to upload your own certificate.
